### ISSUE TITLE
Network in Linux 3.18 doesn't work

### ISSUE THREAD

**Issue by kenorb** [copy/v86#64]
On the [page](http://copy.sh/v86/) it's written:

> With internet access, udhcpc, wget and links.

However I'm testing [Linux 3.18](http://copy.sh/v86/?profile=linux3) and `links` which is installed and it doesn't open anything, secondly `ifconfig` doesn't return any network interfaces. Was this working before?

I've tried also: `/etc/init.d/S40network start` with with no success.

**Comment by copy** [comment]
Run `dhcpcd` first. I'll document this.

**Comment by kenorb** [comment]
Thanks, it works, ignoring some crash:

![screen shot 2015-09-29 at 09 55 52](https://cloud.githubusercontent.com/assets/266306/10159482/4cca40de-6690-11e5-9e59-6db177bdc301.png)

But after that it seems the networking is on. Pity there is no `curl`.

I've tried something like: `wget -qO- ifconfig.co` and there is 'Network is unreachable', however host response to pings. The same with `wget www.google.com`.
Looks like it's trying to IPv6.

![screen shot 2015-09-29 at 10 02 14](https://cloud.githubusercontent.com/assets/266306/10159602/2f14ca9a-6691-11e5-83ab-02728f21085c.png)

Tested in Chrome 45 on OS X.
